src/opera_utils/_dates.py

```diff
@@ -49,6 +49,9 @@ def get_dates(filename: Filename, fmt: str = DATE_FORMAT) -> list[datetime.datet
     path = _get_path_from_gdal_str(filename)
     pattern = _date_format_to_regex(fmt)
     date_list = re.findall(pattern, path.stem)
+    if not date_list:
+        # This is for ionosphere tec files that there is a "." in the file name
+        date_list = re.findall(pattern, path.name)
```

Comment:
Can you add a new test to illustrate what happens in the `test_dates.py` file?

Reply:
sure